### DISCUSSION TITLE
Firefly RK3399-PC-Plus with standard image - boot loop

### DISCUSSION THREAD

**Discussion by belveder79** [johang/sd-card-images#66] [Q&A]
I downloaded all the images and created an sd card to boot Debian or Ubuntu. Unfortunately I end up in a boot loop immediately. 

U-Boot comes up with only some warning about weird SPI bytes, but then it detects USB devices which goes well (no storage devices found), autoboot, then it tries to boot and returns to U-Boot with RST cause.

Am I doing something wrong or is the image simply not working for the Plus version? I did not find a single working version yet...

Thx

**Comment by johang** [comment]
ROC-RK3399-PC Plus is not supported.

**Reply by belveder79** [reply]
that's a pity... just by chance, do you know what the difference is to the standard PC version? Or how to make a version from your build that would probably work on the Plus version?

Your images are the only ones I found so far that get to a boot screen AT ALL - the Firefly images are all crap...

**Reply by johang** [reply]
I don't know the difference. "Plus" seems to be supported by the kernel but not in mainline U-Boot. Post the complete logs from the boot. Maybe there is some hint.

**Comment by belveder79** [comment]
That's the best I can get...
![IMG_7758](https://github.com/johang/sd-card-images/assets/112880/194e1309-c63f-44a7-9d60-f56ad5c79ef1)
